Sources/OpenPass/Request.swift

```diff
@@ -82,3 +82,35 @@ extension Request where ResponseType == OpenPassTokensResponse {
         )
     }
 }
+
+// MARK: - Device Auth
+
+extension Request where ResponseType == DeviceAuthorizationResponse {
```

Comment:
Not related to this code exactly, but wanted to confirm that when we make these requests on tvOS, the suitable headers (we use for metrics), regarding device/platform/etc, are correct? Might be worth sanity checking since we're adding a first proper use case for people running on Apple TVs.